tools/set-leds-default-state.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "utility.hpp"
+
+/**
+ * @brief API to set asserted property.
```

Comment:
set asserted means to set it to true or false ?

Reply:
Updated.